x-pack/plugins/observability/public/pages/rule_details/config.ts

```diff
@@ -33,3 +33,5 @@ export function hasAllPrivilege(rule: InitialRule, ruleType?: RuleType): boolean
 
 export const hasExecuteActionsCapability = (capabilities: Capabilities) =>
   capabilities?.actions?.execute;
+
+export const OBSERVABILITY_SOLUTIONS = ['logs', 'uptime', 'infrastructure', 'apm'];
```

Comment:
We need to avoid that, can you see if we can do it like we refactor the other code

Reply:
@XavierM, could you point out what refactors of _the other code_? And why do we need to avoid this one?

Reply:
Thanks, @maryam-saeidi!